examples/python/TrimEnvelope.py

```diff
@@ -25,12 +25,15 @@
 import math
 
 # Path to JSBSim files, location of the folders "aircraft", "engines" and "systems"
-PATH_TO_JSBSIM_FILES="../../"
+PATH_TO_JSBSIM_FILES="../.."
 
-fdm = jsbsim.FGFDMExec(PATH_TO_JSBSIM_FILES)  
+# Avoid flooding the console with log messages
+jsbsim.FGJSBBase().debug_lvl = 0
+
+fdm = jsbsim.FGFDMExec(PATH_TO_JSBSIM_FILES)
 
 # Load the 737 aircraft model
-fdm.load_model('737') 
```

Comment:
Don't you want to remove my hard-coded 737 and make it a global variable `AIRCRAFT_NAME` like you did in the `PlotAoAVsCAS.py` example?

Reply:
I was assuming that, since there is a single occurrence of the aircraft name in `TrimEnvelope.py`, the user could figure out what to do to change the aircraft name. But for the sake of consistency, it might be better ti use `AIRCRAFT_NAME` in all cases indeed.

Reply:
I'll fix that.

Reply:
This is now fixed.